### DISCUSSION TITLE
How do I make superhot be permanently on?

### DISCUSSION THREAD

**Discussion by Hrudai-Devain** [Pinsplash/halflife2chaos#112] [General]

**Comment by acuifex** [comment]
You probably can use `chaos_time_superhot 99999` with `chaos_test_effect <id>` where the id is from `chaos_print`. Kind of a hacky solution, but it should work.

`chaos 0` to disable automatic effect selection

**Comment by Pinsplash** [comment]
this is generally what i do for videos

**Comment by Hrudai-Devain** [comment]
Thanks!
